apps/laboratory/tests/shared/utils/project.ts

```diff
@@ -39,6 +39,7 @@ export type CustomProjectProperties = {
 }
 
 const SINGLE_ADAPTER_EVM_TESTS = [
+  'metamask.spec.ts',
```

Comment:
Do we need to configure / de-configure sympress for this?

Reply:
1) We need to bring the MM configures back: https://github.com/reown-com/appkit/pull/2636/files#diff-98bf5640bd414ff756b7db8c5a1cc15eca3922044fe80b820259b60491e8c639R108-R126
2) Make sure the current Synpress version is up to date with our Playwright version